src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/validering/BarnetsVilkårValidator.kt

```diff
@@ -23,37 +23,44 @@ class BarnetsVilkårValidator(
         val funksjonelleFeil = mutableListOf<String>()
 
         barna.map { barn ->
-            val vilkårsResultaterForBarn =
-                vilkårsvurdering.personResultater
-                    .flatMap { it.vilkårResultater }
-                    .filter { it.personResultat?.aktør == barn.aktør }
+            val vilkårsResultaterForBarn = vilkårsvurdering.personResultater.flatMap { it.vilkårResultater }.filter { it.personResultat?.aktør == barn.aktør }
```

Comment:
Personlig liker jeg bedre formateringen over da det er lettere å lese på småe skjermer, hvor man kan se innholdet uten å skrolle horisontalt. Syns også det er greit at hver operasjon får sin egen linje, det blir litt lettere å lese. Det er jo mye diskutert hva som er den beste "max line length" for en kodelinje er, hvor f.eks. Google bruker 80. Er det noe man burde ha et sterkere forhold til? Er nokså vanlig å ha mellom 80-120 i prosjekter for å sikre lesbarheten av koden på alle skjermer. I mine tidligere prosjekter har vi så og si alltid brukt 120. Ikke nødvendigvis noe galt i den endringen du gjorde her, bare noen tanker som kom opp.  😊

Reply:
Veldig godt poeng!
Den her ble for noen grunner flikka på av ktlint, så det var ikke meningen å endre på det haha